registry.json

```diff
@@ -4204,5 +4204,35 @@
       "rpc": "https://toncenter.com/api/v2/jsonRPC",
       "documentation": "https://ton.org/docs"
     }
+  },
+  {
+    "id": "neon",
+    "name": "Neon",
+    "coinId": 255022934,
```

Comment:
Why don't you use 245022934?

Reply:
Makes more sense to me now that the numbers need to start from `10000000`, might be worth updating the docs to say this is for chainIds < 10000000

Reply:
coinId=245022934 is not present in `registry.json` yet, so you can use that
> coinId: If own coinID is available, use that. Otherwise, use: 10000000 + chainID, such as 10008217.

Reply:
> coinId=245022934 is not present in `registry.json` yet, so you can use that
> 
> > coinId: If own coinID is available, use that. Otherwise, use: 10000000 + chainID, such as 10008217.

Ah, now I really understand it properly 🤦 sorry! I'm pushing an update now

Thanks for the explanation 🙏